docs/api/state.md

```diff
@@ -1,36 +1,35 @@
 # State
 
-!!! warning "Advanced topic!"
+!!! warning "Stop! Read the interactivity guide first!"
+    **Read the guide on [creating interactive
+    elements](../guides/interactivity.md)** before reading this one!
 
-    This API doc covers reactive state (`mo.state`), an advanced topic.
+!!! warning "Advanced and dangerous topic!"
```

Comment:
"Dangerous" seems a bit aggressive. Maybe like: "you likely don't need this"

Reply:
Yeah, fair enough. I don't know what a "dangerous topic" is. I grew a bit weary of seeing `mo.state()` used in the wild, and really just want to deter all but the most determined from using it. Updated.